src/auto-complete.directive.ts

```diff
@@ -39,6 +39,8 @@ export class NguiAutoCompleteDirective implements OnInit, OnChanges, AfterViewIn
     @Input('header-item-template') public headerItemTemplate = null;
     @Input('ignore-accents') public ignoreAccents: boolean = true;
     @Input('filters') public filters: AutoCompleteFilter[];
+    @Input('itemHeight') public itemHeight = 37;
+    @Input('max-height-top-gap') public maxHeightTopGap = 0;
```

Comment:
@ldiaz can we stay consistent and keep using kebab-case for naming new input bindings being added?

Reply:
updated